libcxx/test/std/time/time.syn/formatter.file_time.pass.cpp

```diff
@@ -821,7 +821,9 @@ static void test_valid_values_date_time() {
   // Use the global locale (fr_FR)
   check(
 // https://sourceware.org/bugzilla/show_bug.cgi?id=24054
-#if defined(__GLIBC__) && __GLIBC__ <= 2 && __GLIBC_MINOR__ < 29
```

Comment:
Powerpc is a CPU architecture. It shouldn't affect the format of times.

Why is the output different on your platform? Is it a different  GLIBC?

Reply:
You are right.  It's not a different GLIBC.  We just configure our locales differently.

Reply:
Is there a different way to identify the system? Or is this valid for all linux based powerpc systems, including the ancient ones?

Reply:
AFAIK this is the way to identify linux powerpc systems.